### ISSUE TITLE
Carousel: aria-hidden is set to true for all carousel items

### ISSUE THREAD

**Issue by KluSe** [primefaces/primeng#17975]
### Describe the bug

The value for aria-hidden is set to true for all p-carousel-items in a carousel. This makes the component inaccessible for assistive technologies like screen readers.

### Pull Request Link

https://github.com/primefaces/primeng/pull/17974

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/github-9rt81grr?file=src%2Fapp%2Fapp.component.html

### Environment

MacOS Sequoia

### Angular version

18 / 19

### PrimeNG version

v19

### Node version

20 / 22

### Steps to reproduce the behavior

1. Check the carousel component in the browser devtools
2. All p-item-carousel elements have aria-hidden="true" set, even those that are visible

### Expected behavior

Only the elements that are out of view should have aria-hidden="true" set

**Comment by mehmetcetin01140** [comment]
fixed in #17974
